dogen/plugins/dist_git.py

```diff
@@ -14,21 +15,58 @@ def info():
     def inject_args(parser):
         parser.add_argument('--dist-git-enable', action='store_true', help='Enables dist-git plugin')
         parser.add_argument('--dist-git-assume-yes', action='store_true', help='Skip interactive mode and answer all question with "yes"')
+        parser.add_argument('--dist-git-repo', help='Repository name to sync with')
+        parser.add_argument('--dist-git-branch', help='Branch name in the repository to sync with')
```

Comment:
this is override for image.yaml? Is it really needed?  It can be used to use 1 image.yaml for two branches, which is weird use case.